ext/scheduler/airflow/dag/dag.py.tmpl

```diff
@@ -49,6 +49,9 @@ default_args = {
     "end_date": datetime.strptime({{ .JobDetails.Schedule.EndDate.Format "2006-01-02T15:04:05" | quote}}, "%Y-%m-%dT%H:%M:%S"),
     {{- end}}
     "weight_rule": WeightRule.ABSOLUTE,
+    {{- if gt .SLAMissDuration 0 }}
+    "sla":timedelta(seconds={{ .SLAMissDuration }}),
```

Comment:
for consistency, let's add space after `"sla":`

Reply:
sure